flow/record/base.py

```diff
@@ -321,7 +318,7 @@ class RecordField:
     typename = None
     type = None
 
-    def __init__(self, name, typename):
+    def __init__(self, name: str, typename: str) -> None:
```

Comment:
```suggestion
    def __init__(self, name: str, typename: str):
```
Return type of `__init__` can be skipped.